cmd/cosign/cli/generate/generate_key_pair.go

```diff
@@ -85,7 +85,12 @@ func GenerateKeyPairCmd(ctx context.Context, kmsVal string, args []string) error
 		return err
 	}
 
-	if cosign.FileExists("cosign.key") {
+	fileExists, err := cosign.FileExists("cosign.key")
+	if err != nil {
+		return err
```

Comment:
how about we wrap the error to give more information?

Reply:
done